index.bs

```diff
@@ -1138,12 +1140,15 @@ To <dfn>canonicalize a sanitizer element with attributes</dfn> a {{SanitizerElem
 
 1. Let |result| be the result of [=canonicalize a sanitizer element=] with |element|.
 1. If |element| is a [=dictionary=]:
-   1. [=list/iterate|For each=] |attribute| in
-      |element|["{{SanitizerElementNamespaceWithAttributes/attributes}}"]:
-      1. [=list/Append=] the result of [=canonicalize a sanitizer attribute=] with |attribute| to |result|["{{SanitizerElementNamespaceWithAttributes/attributes}}"].
-   1. [=list/iterate|For each=] |attribute| in
-      |element|["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"]:
-      1. [=list/Append=] the result of [=canonicalize a sanitizer attribute=] with |attribute| to |result|["{{SanitizerElementNamespaceWithAttributes/removeAttributes}}"].
+   1. If neither |element|["{{SanitizerConfig/attributes}}"] nor
+      |element|["{{SanitizerConfig/removeAttributes}}"] [=map/exist=], then [=map/set=]
+      |result|["{{SanitizerConfig/removeAttributes}}"] to &laquo; &raquo;.
```

Comment:
These are all linking the wrong dictionary I think. They should all be `SanitizerElementNamespaceWithAttributes` instead of `SanitizerConfig`, no?